docs/source/tutorials/image.ipynb

```diff
@@ -22,6 +22,56 @@
     "- Compute a list of potential label errors with cleanlab's `find_label_issues` method.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Already have an sklearn compatible `model`, image `data` and true `labels`? Run the code below to train your `model` and get label issues.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.classification import CleanLearning\n",
+    "\n",
+    "cl = CleanLearning(model)\n",
+    "_ = cl.fit(train_data, labels)\n",
+    "label_issues = cl.get_label_issues()\n",
+    "preds = cl.predict(test_data) # predictions from a version of your model \n",
+    "                              # trained on auto-cleaned data\n",
```

Comment:
Let's delete all the CleanLearning stuff from this quickstart blob. This tutorial never shows how to do anything with CleanLearning, so it is confusing here